dev.salaniLeo.flake.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id" : "dev.salaniLeo.flake",
```

Comment:
`salanileo.dev` doesn't seem to exist.

Reply:
just to clarify, this mean the app-id can't be verified.

Reply:
Sorry if I'm asking obvious questions but how do i solve this?

Reply:
you can use a github linked app-id. See https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
you can use a github linked app-id. See https://docs.flatpak.org/en/latest/conventions.html#application-ids